### PR TITLE
Version 2.1.0

### PR DESCRIPTION
**Pull request by tpkahlon**
# Reference

- Address issue #25

# Improvements

- Add Prettier implementation
- Add ESLint implementation
- Turn off Hot reload option for HTML file as deemed not necessary
- Update documentation for easier access to new dependencies
- Revise layout placement of text in boilerplate

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+  "ignorePatterns": ["dist", "node_modules"],
+  "rules": {
+    // "max-len": [
```
Comment (taniarascia) [inline .eslintrc.json:4]: Looks good, just remove this commented out code?

File: src/index.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // Test import of a JavaScript function
-import {example} from './js/example'
+import example from './js/example'
```
Comment (taniarascia) [inline src/index.js:2]: Is there a reason for changing this to default? Not using default exports helps reduce bugs: https://palantir.github.io/tslint/rules/no-default-export/ But it's not a huge deal.

**Review by taniarascia — COMMENTED** [review]
Just a few comments. Sorry for the delay.

File: src/index.js (hunk 2 above)
Comment (tpkahlon) [reply src/index.js]: Addressed in commit [d633e41](https://github.com/taniarascia/webpack-boilerplate/pull/40/commits/d633e41be30483029eebc44668953f39d199b711).

File: .eslintrc.json (hunk 1 above)
Comment (tpkahlon) [reply .eslintrc.json]: Addressed in commit [d633e41](https://github.com/taniarascia/webpack-boilerplate/pull/40/commits/d633e41be30483029eebc44668953f39d199b711).

**Comment by taniarascia** [comment]
Sorry, I don't think we actually want the `no-default-export` on the ESLint because we want to allow things like `import React from 'react'`, it's just a preferred thing. Otherwise good to go.

**Comment by tpkahlon** [comment]
Agreed, Fixed in commit [c5f0c0b](https://github.com/taniarascia/webpack-boilerplate/pull/40/commits/c5f0c0ba1c47c1c68471ea5c2559d6c6ffe82d60).

**Pull request merged**